lib/iex/lib/iex/autocomplete.ex

```diff
@@ -24,6 +24,14 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  def exports(mod) do
```

Comment:
I think no, `expand/2` is the purpose and the main feature of the `IEx.Autocomplete` module, but `exports/1` is rather a shared helper function so we should emphasise it with `@doc false`.